Documentation/fault-output.rst

```diff
@@ -45,50 +50,51 @@ This output is parametrized by the Elementwise namelist, example:
 printIntervalCriterion
 ~~~~~~~~~~~~~~~~~~~~~~
 
-If printIntervalCriterion = 1, the output is generated every N time steps,
-where N is set by printInterval. This option only works with Global time
-stepping. If printIntervalCriterion = 2, output is generated every
-printtimeinterval_sec.
+
+- When **printIntervalCriterion** = 1, the output is generated every N time steps, where N is defined by **printInterval**. Note that this option is only compatible with Global time stepping. 
+- When **printIntervalCriterion** = 2, output is generated every **printtimeinterval_sec**.
```

Comment:
this printIntervalCriterion has been dropped when porting DR to cpp.
Here are the parameters currently read:
https://github.com/SeisSol/SeisSol/blob/master/src/DynamicRupture/Output/ParametersInitializer.hpp#L87
So you only need to document `printtimeinterval_sec`